src/server/trpc/router/admin.ts

```diff
@@ -68,6 +68,34 @@ export const adminRouter = router({
       });
     }),
 
+  deleteLocation: protectedStaffProcedure

```

Comment:
In the client we initialized this with "trpc.admin.**deleteLocation**.useMutation();" For queries we use `useQuery`.